pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>ariba-plugins</artifactId>
-  <version>1.2.0</version>
+  <version>1.2.1-SNAPSHOT</version>
```

Comment:
The patch versions should not change in the develop branch. You should change this to 1.3.0-SNAPSHOT instead.

Reply:
Done